src/services.yml

```diff
@@ -1,9 +0,0 @@
-services:
-  psr_container:
-    class: RoaveBehatPsrContainer
-    factory: ['Roave\BehatPsrContainer\ContainerFactory', 'createContainerFromIncludedFile']
-    tags:
-      - {'name': 'helper_container.container'}
-    shared: false
-    scope: prototype
```

Comment:
I don't see this `scope` being set anywhere in the PHP configuration, is it not required any more?